### ISSUE TITLE
Fail in catkin build

### ISSUE THREAD

**Issue by mqcmd196** [jsk-ros-pkg/euslime#2]
I failed in catkin build. This is the log.
```bash
yoshiki@yoshiki:~/euslime_ws$ catkin build
-------------------------------------------------------------
Profile:                     default
Extending:             [env] /opt/ros/melodic
Workspace:                   /home/yoshiki/euslime_ws
-------------------------------------------------------------
Build Space:        [exists] /home/yoshiki/euslime_ws/build
Devel Space:        [exists] /home/yoshiki/euslime_ws/devel
Install Space:     [missing] /home/yoshiki/euslime_ws/install
Log Space:         [missing] /home/yoshiki/euslime_ws/logs
Source Space:       [exists] /home/yoshiki/euslime_ws/src
DESTDIR:            [unused] None
-------------------------------------------------------------
Devel Space Layout:          linked
Install Space Layout:        merged
-------------------------------------------------------------
Additional CMake Args:       None
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
Cache Job Environments:      False
-------------------------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
-------------------------------------------------------------
Workspace configuration appears valid.

NOTE: Forcing CMake to run for each package.
-------------------------------------------------------------
[build] Found '1' packages in 0.0 seconds.                                     
Starting  >>> catkin_tools_prebuild                                            
Finished  <<< catkin_tools_prebuild                [ 1.6 seconds ]             
Starting  >>> euslime                                                          
_______________________________________________________________________________
Warnings   << euslime:cmake /home/yoshiki/euslime_ws/logs/euslime/build.cmake.000.log
*** Arguments ['entry_points'] to setup() not supported in catkin devel space in setup.py of euslime
cd /home/yoshiki/euslime_ws/build/euslime; catkin build --get-env euslime | catkin env -si  /usr/bin/cmake /home/yoshiki/euslime_ws/src/euslime --no-warn-unused-cli -DCATKIN_DEVEL_PREFIX=/home/yoshiki/euslime_ws/devel/.private/euslime -DCMAKE_INSTALL_PREFIX=/home/yoshiki/euslime_ws/install; cd -
...............................................................................
_______________________________________________________________________________
Errors     << euslime:install /home/yoshiki/euslime_ws/logs/euslime/build.install.000.log
CMake Error at cmake_install.cmake:157 (file):
  file INSTALL cannot find
  "/home/yoshiki/euslime_ws/src/euslime/slime-repl-ansi-color/README.md".


make: *** [install] Error 1
cd /home/yoshiki/euslime_ws/build/euslime; catkin build --get-env euslime | catkin env -si  /usr/bin/make install; cd -
...............................................................................
Failed     << euslime:install                      [ Exited with code 2 ]      
Failed    <<< euslime                              [ 1.5 seconds ]             
[build] Summary: 1 of 2 packages succeeded.                                    
[build]   Ignored:   None.                                                     
[build]   Warnings:  1 packages succeeded with warnings.                       
[build]   Abandoned: None.                                                     
[build]   Failed:    1 packages failed.                                        
[build] Runtime: 3.1 seconds total.                                            
[build] Note: Workspace packages have changed, please re-source setup files to use them.
```

`yoshiki` is user.

**Comment by Affonso-Gui** [comment]
EDIT: Probably the below is enough 

```bash
cd ~/euslime_ws/src/euslime
git submodule init
git submodule update
cd ~/euslime_ws
catkin build
```

**Comment by mqcmd196** [comment]
Thanks for your replying!
I succeeded in catkin build, but when I execute `emacs -nw -f euslime`, emacs shows the below.
```
--euslisp-program roseus --init-file /home/yoshiki/.euslime/slime-loader.l --port 0 --port-filename /tmp/slime.26491 --color
Traceback (most recent call last):
  File "/usr/local/bin/euslime", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3251, in <module>
    @_call_aside
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'euslime' distribution was not found and is required by the application

Process inferior-lisp exited abnormally with code 1
```

When I tried installing euslime, I used [this](https://github.com/furushchev/euslime/tree/master) repository at first. I couldn't find out whether it has problem or not.

**Comment by Affonso-Gui** [comment]
https://github.com/furushchev/euslime/tree/master is outdated and may be conflicting with the new installation method.

My bet is that the previously pip installed euslime command is clashing with the newly catkin installed one.

1. Try to execute `euslime` in the command prompt and check if the result is similar to the above error.
2. `source /opt/ros/melodic/setup.bash` and then chech `which euslime`. If this points to something like `/usr/local/bin/euslime` then we have a problem.
3. In this case, we need to uninstall the euslime executable. Try going back to https://github.com/furushchev/euslime/tree/master and run `python setup.py develop -u`. When testing in the docker I also had to `sudo rm /usr/local/bin/euslime`.
4. Verify that `which euslime` returns nothing, then source the package and try to run the `euslime` command.
5. If you have something like the following you are probably ready to go
```
[INFO - server:L101] Starting server with encoding utf-8 and color False
[INFO - server:L116] Serving on 0.0.0.0:37983
```

**Comment by mqcmd196** [comment]
I executed your command and euslime works without any troubles! Thanks a lot!!

When I executed `euslime`, the terminal showed the completely same error with https://github.com/jsk-ros-pkg/euslime/issues/2#issuecomment-612794743. `which euslime` also showed `/usr/local/bin/euslime`. So it was true that the new one was crashing with the old one. I'll show what I did.

```bash
cd ~/euslime_dir/euslime
sudo python setup.py develop -u
cd ~/euslime_ws
catkin config --install
catkin build
```

I'll enjoy my euslime life. Thanks!
